system/RESTful/ResourceController.php

```diff
@@ -93,7 +93,7 @@ public function index()
 	 *
 	 * @return array	an array
 	 */
-	public function show($id = null)
```

Comment:
All methods in ResourceController with the `$id=null` arguments must stay. It's intended to be extended and doesn't work if they're missing.